apps/web/package.json

```diff
@@ -17,7 +17,7 @@
     "sass": "node ./scripts/compile-css.js",
     "watch:rollup": "chokidar \"src/client/**/*.js\" --silent -c \"npm run rollup\"",
     "watch:sass": "chokidar \"src/styles/**/*.scss\" --silent -c \"npm run sass\"",
-    "test": "echo 'TOOD: add test script'",
+    "test": "cross-env NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules npx jest",
```

Comment:
Using npx will imply latest version. Might want to keep it using the local bin, so drop the npx.

Reply:
What do you mean by imply latest version? npx will only source the latest package from npm if it it's not present in your package.json dependencies (like, if you wanna run jest without installing it). Running without npx wouldn't guarantee that the local binary is used.